.travis.yml

```diff
@@ -1,25 +1,60 @@
+image: php:7.1.1
 dist: trusty
-sudo: false
+sudo: true
 notifications:
   email: false
-language: node_js
-node_js:
-  - "8"
-  - "6"
-  - "4"
-  - "0.12"
-cache:
-  directories:
-    - node_modules
+language: php
+php:
+  # todo: add test support for other php versions
+  # - '5.6.31'
+  # - '7.0.22'
+  - '7.1.12'
```

Comment:
Reading through https://docs.travis-ci.com/user/languages/php I see that `trusty` support only php@7. I'm ok with this for now, let's not sweat over php@5. Also, if you can drop `sudo: true` and `apt-get install`s, that would be even better - the builds will run much faster.